packages/elements/src/components/ino-segment-button/ino-segment-button.scss

```diff
@@ -64,10 +84,7 @@ $left-right-segment-border: 0 24px 24px 24px;
       background-color: #fff;
     }
   }
-
-  .mdc-button--outlined {
-    @include button.outline-width(1px);
-    @include button.outline-color(var(--segment-button-color));
+  .mdc-button--unelevated {
     @include button.ink-color(var(--segment-button-color));
```

Comment:
In comparison, the font style looks different from the design. 
![image](https://user-images.githubusercontent.com/22963121/186674771-ad25a7a1-a789-49d0-bc1d-f7443f03fee2.png)

- The letters should not be capitalized
- The text should have no letter spacing (at least not as wide)
- The background should be slightly brighter (#E8EDFF)